doc/data_structures.md

```diff
@@ -457,7 +457,7 @@ Example sources_json:
 ```
 
 ## method: indy_build_payment_req
-This API call is handled by LibSovToken build_payment_req_handler. 
+This API call is handled by LibSovToken build_payment_req_handler. Note all payment amounts in outputs must sum to the input utxo value. 
```

Comment:
This is not true unless: fees are 0 for the transaction; or we change the way fees are handled and they become an explicit part of the outputs.